txp/Pos/Txp/Core/Types.hs

```diff
@@ -130,7 +130,7 @@ type TxWitness = Vector TxInWitness
 -- be the same as `txOutValue` of corresponding output.
 type TxOutDistribution = [(StakeholderId, Coin)]
 
--- | Distribution of “fake” stake that follow-the-satoshi would use
+-- | Distribution of "fake" stake that follow-the-satoshi would use
```

Comment:
Don't use quotation markes with Haddock syntax, they need to be escaped like this: `\"foo\"`. Quotation marks (unescaped) are used for module names in Haddock.